packages/react/src/refs.tsx

```diff
@@ -11,13 +12,15 @@ import {
   TemplateEvent,
 } from './template'
 
-export type Store<V = JSONValue> = {
-  get: () => V
-  subscribe: (handler: () => void) => () => void
+type DataSourceInfo = {
+  status: 'connecting' | 'connected' | 'closed' | 'closing' | 'error' | 'undetermined'
```

Comment:
RWS does not report "connecting" and "closing" statuses, they're not part of the spec. Do we still want to keep them? (I suggest let's not)